composer.json

```diff
@@ -50,6 +50,7 @@
         "dms/phpunit-arraysubset-asserts": "^0.3.1",
         "fakerphp/faker": "^1.17",
         "filp/whoops": "^2.14",
+        "mstruebing/editorconfig-checker": "^10.3",
```

Comment:
This change is eventually slightly out of scope of this PR? ;)

(and at the moment does not work inside the docker dev env as editorconfig scans an invalid node_modules config which breaks its parser..)